<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_medicalRecord">
        <property name="text">
         <string>Записаться на прием</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_patientCard">
        <property name="text">
         <string>Посмотреть карту</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_medicalCertificate">
        <property name="text">
         <string>Выдать справку</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QMenu" name="menuCreate_User">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Создать пользователя</string>
     </property>
     <addaction name="actionCreate_Doctor"/>
     <addaction name="actionCreate_Patient"/>
     <addaction name="actionCreate_Registrator_2"/>
    </widget>
    <addaction name="menuCreate_User"/>
    <addaction name="actionSpecialty"/>
    <addaction name="action_patientCard"/>
    <addaction name="separator"/>
    <addaction name="actionLogOut"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLogOut">
   <property name="text">
    <string>Выход из профиля</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionCreate_Patient">
   <property name="text">
    <string>Пациент</string>
   </property>
  </action>
  <action name="actionCreate_Registrator">
   <property name="text">
    <string>Create Registrator</string>
   </property>
  </action>
  <action name="actionCreate_Doctor">
   <property name="text">
    <string>Доктор</string>
   </property>
  </action>
  <action name="actionEdit_Prifil">
   <property name="text">
    <string>Edit Prifile</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCreate_Registrator_3">
   <property name="text">
    <string>Create Registrator</string>
   </property>
  </action>
  <action name="actionCreate_Registrator_2">
   <property name="text">
    <string>Регистратор</string>
   </property>
  </action>
  <action name="actionSpecialty">
   <property name="text">
    <string>Добавить\Удалить специальность</string>
   </property>
  </action>
  <action name="action_patientCard">
   <property name="text">
    <string>Посмотреть свою карту</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
